pipelines/hacbs/hacbs-test.yaml

```diff
@@ -22,11 +22,23 @@
       name: sanity-label-check
     params:
     - name: IMAGE_URL
+      value: $(params.output-image)   
+- op: add
+  path: /spec/tasks/-
+  value:
+    name: get-clair-results
+    runAfter:
+      - build-container
+    taskRef:
+      name: get-clair-scan
+    params:
+    - name: sha
```

Comment:
`    invalid input params for task get-clair-scan: missing values for these params which have no default values: [SHA PULLSPEC]` -> must be uppercase

Reply:
Thanks for pointing that out, fixed.